manim/scene/scene.py

```diff
@@ -1008,6 +1008,22 @@ def play(
             All other keywords are passed to the renderer.
 
         """
+        # Make sure this is running on the main thread
+        if threading.current_thread().name != "MainThread":
+            self.queue.put(
+                (
+                    "play",
+                    args,
+                    kwargs
+                    | {
```

Comment:
I think it would be better to use the conventional update method for dicts, the `|` syntax was only introduced with like Python 3.9 and up, no? Without a corresponding `__future__` import, this will break Manim on Python 3.7 + 3.8.